platform/lambda/lambda.go

```diff
@@ -393,12 +382,12 @@ func (p *Platform) deploy(region, stage string) (version string, err error) {
 
 	defer func() {
 		fields["duration"] = time.Since(start)
-		fields["version"] = version
```

Comment:
need to keep this part, I'll tweak it back to the old semantics, not seeing a reason why we need the CreateStack within deploy()

Reply:
You're right about that. I'm changing back to the old semantics.